tailcfg/tailcfg.go

```diff
@@ -930,7 +930,7 @@ func (v HostinfoView) TailscaleSSHEnabled() bool { return v.ж.TailscaleSSHEnabl
 // TailscaleFunnelEnabled reports whether or not this node has explicitly
 // enabled Funnel.
 func (hi *Hostinfo) TailscaleFunnelEnabled() bool {
-	return hi != nil && hi.WireIngress
+	return hi != nil && hi.IngressEnabled
```

Comment:
As a side note, I imagine it could be nice if the hostinfo contained the capver as well. Then this method could just return either `WireIngress` or `IngressEnabled` and the caller wouldn't need to check capver